aws-android-sdk-s3/src/test/java/com/amazonaws/services/s3/model/ObjectMetadataTest.java

```diff
@@ -77,6 +79,7 @@ public void cloneNonEmpty() {
         from.setHeader(Headers.CONTENT_RANGE, "/9999");
         from.setHeader(Headers.S3_VERSION_ID, "versionid");
         from.setHeader(Headers.ETAG, "etag");
+        from.setStorageClass(StorageClass.ReducedRedundancy);
```

Comment:
Unfortunately, there is no method called setStorageClass. Instead, can you use setHeader with Headers.STORAGE_CLASS?

```
from.setHeader(Headers.STORAGE_CLASS, StorageClass.ReducedRedundancy.toString());
```